test/uk/gov/hmrc/plasticpackagingtax/registration/controllers/group/GroupMemberGrsControllerSpec.scala

```diff
@@ -120,10 +140,11 @@ class GroupMemberGrsControllerSpec extends ControllerSpec {
                                incorporationRegistrationDetails
           )
         )
-        mockRegistrationFind(registeredLimitedCompany)
+        val registration = registrationWithSelectedGroupMember(OrgType.UK_COMPANY)
```

Comment:
Possibly irrelevant, but do we need a similar test for `OrgType.OVERSEAS_COMPANY_UK_BRANCH`?

Reply:
I don't think so.  The "duplicate" and "already registered" checks are not dependant on the organisation type.
But I'll add the tests just to make sure.

Reply:
Refactored spec to include this (and others)